packages/sit-onyx/src/components/OnyxAccordion/types.ts

```diff
@@ -2,7 +2,11 @@ import type { SkeletonInjected } from "src/composables/useSkeletonState";
 import type { InjectionKey, Ref } from "vue";
 import type { DensityProp } from "../../composables/density";
 
-export type OnyxAccordionProps = DensityProp & {
+export type OnyxAccordionProps<TValue extends PropertyKey> = DensityProp & {
+  /**
+   * Currently opened items. Will include the `value` property of the nested `OnyxAccordionItems`.
+   */
+  modelValue?: TValue[];
```

Comment:
What about using an object based API? It is easier and more intuitive to use. Also it could be extended without breaking changes :) 

```suggestion
  modelValue?: { [key: TValue]?: Nullable<boolean> };
```

Reply:
I'd like to stick to the array API since the usage/typing for the project is easier imho and also all other libraries I found use an array for the open items (Element Plus, Nuxt UI, Prime Vue etc.) :)